versions/3.0.md

```diff
@@ -51,7 +51,12 @@ The HTTP Status Codes are used to indicate the status of the executed operation.
 ### Format
 
 The files describing the RESTful API in accordance with the OpenAPI Specification are represented as JSON objects and conform to the JSON standards. YAML, being a superset of JSON, can be used as well to 
-represent a OAS (OpenAPI Specification) file.
+represent a OAS (OpenAPI Specification) file. For YAML, version `1.1` is supported for OAS documents.
+
+As YAML has a superset of features, the following YAML features are NOT supported by the OAS:
+
+* YAML Merge. Consider using JSON Pointers instead
+* Numeric keys. All numeric keys should be quoted
```

Comment:
Better add the clarification than link to an extra document we produced ourselves. It's a bit too meta ;)